index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Home &mdash; Leilani's Personal Profile Site</title>
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+</head>
+
+<body style="background: linear-gradient(to bottom, #33ccff 0%, #ff99cc 100%)">
+    <div class="topnav">
+        <a class="active" href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="portfolio.html">Portfolio</a>
+        <a href="hobbies.html">Hobbies</a>
+    </div>
+</body>
+
+<div id="header">
+    <h1 style="color:white">
+        <center>Hello, I'm Leilani Allen.</center>
+    </h1>
+    <h2 style="color:white">
+        <center>Full Stack Developer</center>
+    </h2>
+
+
+
+    <div id="content">
+        <center><img src="/Users/leilani/Developer/Ada/Projects/personal-portfolio-site/images/Leilani_Profile_Pic.jpeg"
+            alt="Profile Picture"></center>
```

Comment:
Please use **relative** path when referring to another resource/file, especially images. Currently, you're using **absolute** path which finds the path to a file starting from the root of your local machine. When you push files into a remote repo like github and deploy, other folx who clone your project will not have the same path as the one you list here as "/Users/leilani/Developer/....etc." 

**Relative** path refers to the path to the specific file/resource starting from the **project root*** so instead this should be turned into:

```suggestion
        <center><img src="images/Leilani_Profile_Pic.jpeg" alt="Profile Picture"></center>
```
OR

```suggestion
    <div id="content">
        <center><img src="./images/Leilani_Profile_Pic.jpeg"
            alt="Profile Picture"></center>
```

The `.` in front of the path just indicates the `root`.

>  ./images/Leilani_Profile_Pic.jpeg

So if we were to verbalize the path above it would be from the root `.` in the folder `/images/` get the file `Leilani_Profile_Pic.jpeg`. Omitting the `.` works just fine as well so the path can also be

> /images/Leilani_Profile_Pic.jpeg